### DISCUSSION TITLE
Documents of pyuvm

### DISCUSSION THREAD

**Discussion by MFJI** [pyuvm/pyuvm#198] [Q&A] [answered]
Hello,

Is there any other document of pyuvm except the readme file here?

Thank you.

**Comment by raysalemi** [comment, marked as answer]
Hi,

I wrote this book
https://www.amazon.com/Python-RTL-Verification-complete-course/dp/B0BCZ1JM3P/ref=mp_s_a_1_1?crid=344ZRIDK5L0QR&dib=eyJ2IjoiMSJ9.KJ9GwajXZCVC11a3NoYGmA.Wi9KXvGnN_cCuJW8mlBhFjCNyCmfZWuFQeGHMLdZsJE&dib_tag=se&keywords=python+for+rtl+verification&qid=1712886433&sprefix=puthon+for+rtl%2Caps%2C92&sr=8-1


Ray

**Comment by raysalemi** [comment]
Hi,

I wrote [this book](https://www.amazon.com/Python-RTL-Verification-complete-course/dp/B0BCZ1JM3P/ref=mp_s_a_1_1?crid=344ZRIDK5L0QR&dib=eyJ2IjoiMSJ9.KJ9GwajXZCVC11a3NoYGmA.Wi9KXvGnN_cCuJW8mlBhFjCNyCmfZWuFQeGHMLdZsJE&dib_tag=se&keywords=python+for+rtl+verification&qid=1712886433&sprefix=puthon+for+rtl%2Caps%2C92&sr=8-1).
